main/src/library/MutList.flix

```diff
@@ -165,7 +424,20 @@ namespace MutList {
     ///
     /// If the given index `i` exceeds the length of the mutable list, the element is inserted at the last position.
     ///
-    pub def insert(v: mut MutList[a], x: a, i: Int32): Unit = ???
+    pub def insert(x: a, i: Int32, v: mut MutList[a]): Unit & Impure =
```

Comment:
Does this function really *insert* an element at index i? It seems like it can *insert* at the end or update in the middle-- which seems fishy.

Reply:
I would have expect two usages of updateSequence.

Reply:
But I could be wrong :)

Reply:
Ahh I did not realize that was the desired behaviour. You're right.